t/sentmap.c

```diff
@@ -100,3 +100,99 @@ void test_sentmap(void)
 
     quicly_sentmap_dispose(&map);
 }
+
+
+static void test_late_ack(void)
+{
+    quicly_sentmap_t map;
+    quicly_sentmap_iter_t iter;
+    const quicly_sent_packet_t *sent;
+
+    on_acked_callcnt = 0;
+    on_acked_ackcnt = 0;
+
+    quicly_sentmap_init(&map);
+
+    /* commit pn 1, 2 */
+    quicly_sentmap_prepare(&map, 1, 0, 0);
+    quicly_sentmap_allocate(&map, on_acked);
+    quicly_sentmap_commit(&map, 10);
+    quicly_sentmap_prepare(&map, 2, 0, 0);
```

Comment:
```suggestion
    quicly_sentmap_prepare(&map, 2, 0, QUICLY_EPOCH_INITIAL);
```

Reply:
The constants are not available in this PR.